ts/package.json

```diff
@@ -41,6 +41,7 @@
     ]
   },
   "dependencies": {
+    "@cosmjs/math": "^0.33.0",
```

Comment:
need to check if this version is compatible with sdk-47.
go implementation uses math package of version v1.3.0, which is not latest

Reply:
do you have recommendation how to check it? All test cases from cosmos testdata are passed